docs/specs/Mountain State Adventure Destination/SPEC-09-adventure-hero-component/spec.md

```
@@ -0,0 +1,319 @@
+- Content Collections schema - Adventure type definition
+
+### External
+
+- `astro:assets` - Image optimization
+- `vitest` + `vitest-axe` - Testing
+
+---
+
+## Clarifications
+
+### Session 2025-12-26
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)